lib/cli/commands/build.js

```diff
@@ -52,28 +53,31 @@ optimized output compatible with Node 6.
     });
   }
 
-  build(dirs = this.dirs) {
-    if (!dirs) {
-      dirs = this.dirs = [ 'app', 'config', 'blueprints' ];
+  build(globs = this.globs) {
+    if (!globs) {
+      globs = this.globs = [ 'app/**/*', 'config/**/*', 'blueprints/*/index.js' ];
```

Comment:
So it looks like we also need globs for files that aren't transformed, since we probably also want to keep the `files` dir in the same place and not have to look in two places.